test/integration_test/operator-test-pod-template.yaml

```diff
@@ -64,6 +64,7 @@ spec:
     - -is-gke=IS_GKE
     - -cloud-provider=CLOUD_PROVIDER
     - -px-upgrade-hops-url-list=PX_UPGRADE_HOPS_URL_LIST
+    - -operator-image-tag=OPERATOR_IMAGE_TAG
```

Comment:
This param is for the tag only to use for the Openshift Marketplace operator image registry, this has to be a full tag, anything custom won't work and it might create confusion, I have on my plate to implement deploying operator in our tests as well, I will think of a way to do that once I actually get to that method. I want to leave this as is for now